### ISSUE TITLE
SharePoint (beta): list of drives should give more information

### ISSUE THREAD

**Issue by Tekki** [OneDrive/onedrive-api-docs#569]
A call to `https://graph.microsoft.com/beta/sharepoint/sites/{site_id}/drives` returns a list of 4 drives with driveType, id and owner (and sometimes quota). It is not possible to distinguish these drives. Even `https://graph.microsoft.com/beta/sharepoint/sites/{site_id}/drives/{drive_id}` doesn't give more information. Wouldn't it be helpful if we had a property similar to 'specialFolder' that tells us what is the function of a drive?

**Comment by rgregg** [comment]
We're working on adding the `name` property to `drive` so you can see the name of the document library.

**Comment by Tekki** [comment]
The same as #568, this seems to be solved in the v1.0 API.
```
{
    "@odata.context": "https://graph.microsoft.com/v1.0/$metadata#drives",
    "value": [
        {
            ...
            "name": "Dokumente",
            "webUrl": "...",
            "driveType": "documentLibrary",
            ...
```

**Comment by rgregg** [comment]
That's right, this is resolved with some changes we made to Microsoft Graph last week.
